src/main/scala/org/openhorizon/exchangeapi/table/user/UserRow.scala

```diff
@@ -10,6 +10,7 @@ import java.util.UUID
 
 case class UserRow(createdAt: java.sql.Timestamp,
                    email: Option[String] = None,
+                   externalId: Option[String] = None, 
```

Comment:
Have to move this to the end to satisfy the implicit `.mapTo` function on the end of your `def * ...`